crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -2249,17 +2358,20 @@ type StakeSubsidy {
 	decreaseRate: Int
 }
 
+"""
+Represents a staked SUI token.
```

Comment:
suggestion: An 0x3::staking_pool::StakedSui object that provides information on the status etc etc

Although this is redundant info from its fields, so maybe just "An 0x3::staking_pool::StakedSui object"